src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -431,6 +431,9 @@ Copyright (c) .NET Foundation. All rights reserved.
          References="@(ReferencePath)"
          KeyContainer="$(KeyContainerName)"
          KeyFile="$(KeyOriginatorFile)"
+         NoConfig="true"
```

Comment:
This one does not need to be $(NoConfig), to be settable?

Reply:
NoConfig=true is hard-coded for main assembly compilation too: https://github.com/dotnet/roslyn/blob/5e30fbc11bf2f4c4857616fdb0d70dbe364e5094/src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets#L118